### PR TITLE
DO NOT MERGE: firestore comparison CL

### PR DESCRIPTION
**Pull request by noahdietz**

### CODE REVIEW

File: google-cloud-clients/google-cloud-firestore/src/main/java/com/google/cloud/firestore/v1/FirestoreClient.java [hunk 1]
```diff
@@ -97,13 +96,13 @@
  * methods:
  *
  * <ol>
- *   <li>A "flattened" method. With this type of method, the fields of the request type have been
+ *   <li> A "flattened" method. With this type of method, the fields of the request type have been
```
Comment (schmidt-sebastian) [inline google-cloud-clients/google-cloud-firestore/src/main/java/com/google/cloud/firestore/v1/FirestoreClient.java]: I assume this is a bug in the generator and not in your CL. We should ideally remove the leading spaces.

File: google-cloud-clients/google-cloud-firestore/src/test/java/com/google/cloud/firestore/v1/FirestoreClientTest.java [hunk 2]
```diff
@@ -156,15 +149,15 @@ public void deleteDocumentTest() {
     Empty expectedResponse = Empty.newBuilder().build();
     mockFirestore.addResponse(expectedResponse);
 
-    AnyPathName name = AnyPathName.of("[PROJECT]", "[DATABASE]", "[DOCUMENT]", "[ANY_PATH]");
+    String name = "name3373707";
```
Comment (schmidt-sebastian) [inline google-cloud-clients/google-cloud-firestore/src/test/java/com/google/cloud/firestore/v1/FirestoreClientTest.java]: Where does this name come from? We should make this a valid document name (which includes a collection).

File: google-cloud-clients/google-cloud-firestore/src/test/java/com/google/cloud/firestore/v1/FirestoreClientTest.java [hunk 3]
```diff
@@ -198,11 +191,11 @@ public void batchGetDocumentsTest() throws Exception {
             .setTransaction(transaction)
             .build();
     mockFirestore.addResponse(expectedResponse);
-    String formattedDatabase = DatabaseRootName.format("[PROJECT]", "[DATABASE]");
+    String database = "database1789464955";
```
Comment (schmidt-sebastian) [inline google-cloud-clients/google-cloud-firestore/src/test/java/com/google/cloud/firestore/v1/FirestoreClientTest.java]: These names are also not formatted correctly. Are the automatically generated?

File: google-cloud-clients/google-cloud-firestore/src/main/java/com/google/cloud/firestore/v1/FirestoreClient.java (hunk 1 above)
Comment (noahdietz) [reply google-cloud-clients/google-cloud-firestore/src/main/java/com/google/cloud/firestore/v1/FirestoreClient.java]: Correct.

**Pull request closed without merging**